cmd/telemetrygen/internal/logs/config.go

```diff
@@ -24,4 +26,6 @@ func (c *Config) Flags(fs *pflag.FlagSet) {
 
 	fs.IntVar(&c.NumLogs, "logs", 1, "Number of logs to generate in each worker (ignored if duration is provided)")
 	fs.StringVar(&c.Body, "body", "the message", "Body of the log")
+	fs.StringVar(&c.SeverityText, "severity-text", "Info", "Severity of the log, one of: Trace, Debug, Info, Warn, Error, Fatal")
```

Comment:
This can have an arbitrary value according to the spec https://opentelemetry.io/docs/specs/otel/logs/data-model/#field-severitytext
```suggestion
	fs.StringVar(&c.SeverityText, "severity-text", "Info", "Severity text of the log")
```